src/Core/Domain/Product/CommandHandler/UpdateProductOptionsHandlerInterface.php

```diff
@@ -26,15 +26,15 @@
 
 namespace PrestaShop\PrestaShop\Core\Domain\Product\CommandHandler;
 
-use PrestaShop\PrestaShop\Core\Domain\Product\Command\UpdateProductDescriptionCommand;
+use PrestaShop\PrestaShop\Core\Domain\Product\Command\UpdateProductOptionsCommand;
 
 /**
- * Defines contract for UpdateProductDescriptionHandler
+ * Defines contract for UpdateProductOptionsHandler
  */
-interface UpdateProductDescriptionHandlerInterface
+interface UpdateProductOptionsHandlerInterface
```

Comment:
Weird 😅 
A forgotten file I guess?

Reply:
yes, the one I merged into "basicInformation" command. Forgot to delete one apparently :sweat_smile: